Cooper/ClientApp/src.v2/app/services/index.ts

```diff
@@ -0,0 +1,4 @@
+
+export {AuthentificationService} from './authentification/authentification.service';
+
+export {UserService} from './model-services/user.service';
```

Comment:
Actually, you don't need for each service folder

Reply:
This is comfortable way to operate services through folders. We have a lot of models on server side. The time will come when we need to add appropriate services to our client. It's comfortable way to group these services. For each another service we have two files. In order to make our orientation easier we group two relative files in one folder.